Libs/Gtk3/Classes/AccelGroup.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Gdk;
+using GObject;
+using Object = GObject.Object;
+
+namespace Gtk
+{
+    public partial class AccelGroup
+    {
+        #region Properties
+
+        #region IsLockedProperty
+
+        /// <summary>
+        /// Property descriptor for the <see cref="IsLocked"/> property.
+        /// </summary>
+        public static readonly Property<bool> IsLockedProperty = Property<bool>.Wrap<AccelGroup>(
+            Native.IsLockedProperty,
+            nameof(IsLocked),
+            get: (o) => o.IsLocked
+        );
+
+        public bool IsLocked
+        {
+            get => GetProperty(IsLockedProperty);
+        }
+
+        #endregion
+
+        #region ModifierMaskProperty
+
+        /// <summary>
+        /// Property descriptor for the <see cref="ModifierMask"/> property.
+        /// </summary>
+        public static readonly Property<Gdk.ModifierType> ModifierMaskProperty =
+            Property<Gdk.ModifierType>.Wrap<AccelGroup>(
+                Native.ModifierMaskProperty,
+                nameof(ModifierMask),
+                get: (o) => o.ModifierMask
+            );
+        
+        public Gdk.ModifierType ModifierMask
+        {
+            get => GetProperty(ModifierMaskProperty);
+        }
+
+        #endregion
+
+        #endregion
+
+        #region Methods
+
+        public static AccelGroup FromAccelClosure(AccelGroupActivate closure) =>
+            WrapPointerAs<AccelGroup>(Native.from_accel_closure(Marshal.GetFunctionPointerForDelegate(closure)));
+        
+        public void Connect
+        (
+            uint accelKey,
+            ModifierType accelMods,
+            AccelFlags accelFlags,
+            AccelGroupActivate closure
+        )
+        {
+            IntPtr ptr = Marshal.GetFunctionPointerForDelegate(closure);
```

Comment:
Oh actually I wonder if this is working (if we ignore the garbage collection problem)?

@na2axl did you test this?

If it is working I wonder what we should prefer, the native closure or the marshalling way? Any thoughts? Should we try to avoid native closures to keep complexity low?

Reply:
Not tested yet... But yes I remember now that this marshalling got some times garbage collected. To use `ClosureHelper` I have to made it public or allow the `Gtk3` project uses internals from `GObject` since it's an internal class.
But can we use function pointers of C# 9 here ?

Reply:
As this requires more evaluation which is out of scope of this PR I would suggest that we just comment out the code and add a "Todo" flag including a reference to a new issue?

Reply:
The code has been commented and the issue as been created (#119)